newsAppView.test.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const NewsAppView = require('./newsAppView');
+const fs = require('fs');
+
+
+describe("NewsAppView", () => {
+  it("should be able to display articles from the model", () => {
+    document.body.innerHTML = fs.readFileSync('./index.html');
+    let view = new NewsAppView(mockModel, mockApi);
+    view.displayArticles();
+    expect(document.querySelectorAll(".article").length).toBe(1);
+  })
+  it("should be able to show images on the page", () => {
+    document.body.innerHTML = fs.readFileSync('./index.html');
+    let view = new NewsAppView(mockModel, mockApi);
+    view.displayArticles();
+    expect(document.querySelectorAll("img").length).toBe(1);
+  })
+  it("adds a summary on click", () => {
+    document.body.innerHTML = fs.readFileSync('./index.html');
+    let view = new NewsAppView(mockModel, mockApi);
+    view.displayArticles();
+    let article = document.querySelector(".article");
+    article.click();
+    expect(document.querySelectorAll(".summary").length).toBe(1);
+  })
+  it("should be able to clear summaries", () => {
+    document.body.innerHTML = fs.readFileSync('./index.html');
+    let view = new NewsAppView(mockModel, mockApi);
+    view.displayArticles();
+    let article = document.querySelector(".article");
+    article.click();
+    view.clearSummaries();
+    expect(document.querySelectorAll(".summary").length).toBe(0);
+  })
+  it("summaries should only open once", () => {
```

Comment:
Great use of tests, very clear to read and they have a lot of coverage over the methods used